zapcore/console_encoder.go

```diff
@@ -103,52 +102,52 @@ func (c consoleEncoder) EncodeEntry(ent Entry, fields []Field) ([]byte, error) {
 	// If there's no stacktrace key, honor that; this allows users to force
 	// single-line output.
 	if ent.Stack != "" && c.StacktraceKey != "" {
-		line.WriteByte('\n')
-		line.WriteString(ent.Stack)
+		line.AppendByte('\n')
```

Comment:
It's actually the `WriteByte` method that gets flagged: https://golang.org/cmd/vet/#hdr-Methods

The returned error is particularly irritating because it then triggers `errcheck` if you don't use it.